server/webapp/WEB-INF/rails.new/spec/presenters/api_v2/config/environment_config_representer_spec.rb

```diff
@@ -40,6 +40,16 @@
                                 })
     end
 
+    it 'renders appropriate href url for remote environments' do
+      presenter = ApiV2::Config::EnvironmentConfigRepresenter.new(get_environment_config, true)
+      actual_json = presenter.to_hash(url_builder: UrlBuilder.new)
+
+      expect(actual_json).to have_links(:self, :find, :doc)
+      expect(actual_json).to have_link(:find).with_url('http://test.host/api/admin/environments/:environment_name/withremote')
```

Comment:
Ideally `withremote` should just be a query param `/admin/environments/:environmane_name?remote=true`